threema-connector-webtest/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>threema.connector.webtest</groupId>
+  <artifactId>threema-connector-webtest</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
```

Comment:
streamline the version: all <version> 's in your pom should contain the exact same number ... so the build can raise them at once for you when releasing and preparing the next development cycle.

Reply:
changed version to match other pom.xml